### ISSUE TITLE
let_var_whitespace should not trigger for @objc

### ISSUE THREAD

**Issue by fcy** [realm/SwiftLint#1770]
I like the new `let_var_whitespace ` rule but I think it shouldn't trigger for `@objc` because it applies to that member. E.g. the following snippet triggers a violation:

```
    @objc(status)
    var statusObjc: String {
//...
```

What do you think?

**Comment by Uncommon** [comment]
Yeah, that should be easy to fix. I got it to handle a couple of other @ keywords but didn't think of `@objc`.
